### DISCUSSION TITLE
How can I resolve the Pydantic warnings?

### DISCUSSION THREAD

**Discussion by harshalizode** [dymmond/esmerald#372] [Q&A] [answered]
Hello,
I have a question: when I start the application, it will raise the following Pydantic model warnings:

![pydantic-warnings-esmeralds](https://github.com/user-attachments/assets/182fb252-1c2b-407b-b14b-91b2f28be005)
How can we resolve this???

**Comment by tarsil** [comment]
@harshalizode actually this is coming from pydantic and there is already something in place to address this. You don't need to worry, it's not an error and won't cause you any harm. We are doing some internal updates as well to see if we can disable this by default.

EDIT: this will be addressed in Mongoz

**Reply by harshalizode** [reply, marked as answer]
Okay, Thank you.

**Reply by tarsil** [reply]
Not a problem, again, not an error. Just a pydantic warning for this

**Reply by harshalizode** [reply]
@tarsil this issue is not getting addressed yet. We are still getting the Pydantic warnings while running the project.

**Reply by tarsil** [reply]
> @tarsil this issue is not getting addressed yet. We are still getting the Pydantic warnings while running the project.

It's not an issue at all. It's a warning. We will be checking that to be removed later but it's not an error at all

**Reply by harshalizode** [reply]
Okay.

**Reply by devkral** [reply]
shadowing means: an attribute of one base class which is also a pydantic model is overwritten.
